data/strings/types_strings.txt

```diff
@@ -40516,6 +40516,7 @@ fi = Laskettelurinne
 fr = Piste de ski alpin
 gl = Pista de esquí alpino
 zh-Hans = 高级速降滑雪区
+zh-Hant = 下坡滑雪道
```

Comment:
Area should have exactly the same translation as non-area. All translations in this block and other similar area blocks should be removed.